### PR TITLE
github corners support

### PR DESCRIPTION
**Pull request by smidm**
support https://github.com/tholman/github-corners, inspired by https://github.com/jsliang/pelican-fresh git ribbon feature

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -182,5 +182,6 @@
   {% include "partial/jsonld.html" %}
 
   {% block additional_js %}{% endblock %}
+  {% include 'partial/github.html' %}
```
Comment (alexandrevicenzi) [inline templates/base.html]: Following other plugins the if goes here so we avoid including a file that wont be used.

File: templates/partial/github.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{% if GITHUB_URL %}
```
Comment (alexandrevicenzi) [inline templates/partial/github.html]: Place this if in the "include" section.

**Comment by smidm** [comment]
fixed

**Comment by alexandrevicenzi** [comment]
Should we use `GITHUB_CORNER_URL`?

It's more clear to know for what is.

**Comment by smidm** [comment]
ok for me, should I update?

**Comment by alexandrevicenzi** [comment]
Yes.

**Comment by smidm** [comment]
ok?

**Comment by alexandrevicenzi** [comment]
@smidm Thanks.

**Pull request merged**